pkg/volume/flexvolume/driver-call.go

```diff
@@ -124,7 +128,35 @@ func (dc *DriverCall) Run() (*DriverStatus, error) {
 		defer timer.Stop()
 	}
 
-	output, execErr := cmd.CombinedOutput()
```

Comment:
@mtanino Yes, it is possible. In this case we will need to wait a process to use its stderr output. My idea was to log data from stderr as soon as we recieve them. It is usefull when a target process is running for a long time or if it hangs for some reason. What do you think about that?